CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 * [ENHANCEMENT] Log: Avoid expensive log.Valuer evaluation for disallowed levels. #5297
 * [ENHANCEMENT] Improving Performance on the API Gzip Handler. #5347
 * [ENHANCEMENT] Emit querier `max_concurrent` as a metric. #5362
+* [ENHANCEMENT] Added Query String, Start, End, Step, Ts to Querier Stats.
```

Comment:
Can we add the PR number at the end of the entry like other entries? Thanks!

Reply:
sure. Will add.